src/patternfly/components/FormControls/examples/form-controls-select-example.hbs

```diff
@@ -16,4 +16,14 @@
     <option value="Option 3">The third option</option>
     <option value="Option 4">The fourth option</option>
   </optgroup>
+{{/form-controls}}
+{{#> form-controls controlType="select" pf-c-form-control--modifier="pf-m-error" aria-invalid="true" aria-errormessage="incorrect text" id="select2" name="selectexample2" aria-label="select group example"}}
+  <optgroup label="Group 1">
```

Comment:
In conjunction with styling, `:valid` and `:invalid`, dropping `pf-m-error` and `aria-invalid` here allows the select to react to valid input. `aria-errormessage` stays put.
```
{{#> form-controls controlType="select" required="true" aria-errormessage="incorrect text" id="select2" name="selectexample2" aria-label="select group example"}}
```